src/css/components/_captions-settings.scss

```diff
@@ -21,6 +21,36 @@
   vertical-align: bottom;
 }
 
+// code that will only run if CSS Grid is supported by the browser
+@supports (display: grid) {
+  .vjs-text-track-settings .vjs-modal-dialog-content {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr auto;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-colors {
+    display: block;
+    grid-column: 1;
+    grid-row: 1;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-font {
+    grid-column: 2;
+    grid-row: 1;
+  }
+
+  .vjs-text-track-settings .vjs-track-settings-controls {
+    grid-column: 2;
+    grid-row: 2;
+    text-align: unset;
+    vertical-align: unset;
+  }
+  .vjs-text-track-settings fieldset span.vjs-opacity {
+    margin-left: unset;
```

Comment:
unsetting the margin makes the dropdowns too close. Should probably be removed too.

Reply:
just updated it to give the margin-right on the <select> and keep the left margin unset. this way it will have a gap between them still and be aligned vertically on small players